tests/012_async.test

```diff
@@ -0,0 +1,3 @@
+title: async...await
```

Comment:
Let's leave out the title. The file name is good enough. 

1) It's more minimal
2) It opens up the possibility that only the title is displayed - which take greping to associate with the file ... the filename immediately resolves